tox.ini

```diff
@@ -1,13 +1,14 @@
 [tox]
 isolated_build = true
-envlist = py37-django22,py37-django32,{38,39,310}-django{22,32,40,41},report
+envlist = py37-django22,py37-django32,py{38,39,310}-django{22,32,40,41,42},report
```

Comment:
I noticed the "py" was missing - this might need propagating back to the project template?